testing/scripts/test_helpdesk_pw.py

```
@@ -3499,6 +3501,87 @@ def test_56_dark_mode(page: Page):
+    check("toast #333: toast is visible after settings save",
+          toast.count() > 0,
+          "toast element with .swh-toast--visible not found")
+
+    # URL should not contain swh_notice=saved after JS cleans it.
+    check("toast #334: URL param swh_notice=saved stripped by JS",
+          'swh_notice=saved' not in page.url,
+          f"URL still contains swh_notice=saved: {page.url!r}")
+
+    # Toast auto-dismisses within 6 seconds (4s timer + transition).
+    page.locator('.swh-toast.swh-toast--visible').wait_for(state='hidden', timeout=6000)
+    check("toast #333: toast auto-dismissed within 6s", True)
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)